src/CompiledOptions.h

```diff
@@ -115,6 +122,13 @@ class CmdOptions {
         return num_jobs;
     }
 
+    /**
+     * get index of stratum to be executed
+     */
+    size_t& getStratumIndex() {
```

Comment:
Should this be a const ref, or perhaps a plain size_t?

Reply:
Will change to a plain size_t.

Reply:
Fixed.